auth/src/main/res/values/styles.xml

```diff
@@ -31,33 +31,6 @@
         <item name="android:fontFamily">sans-serif</item>
     </style>
 
-    <style name="FirebaseUI.Text.T01">
```

Comment:
Can we just tell lint to ignore the unused styles?  It's sort of useful to have these here so that when we make new screens we don't have to pick new magic numbers.